packages/yumemi_lints/lib/src/command_services/update_command_service.dart

```diff
@@ -0,0 +1,169 @@
+      return ExitStatus.error;
+    }
+  }
+
+  ExitStatus _updateLintRule(ProjectType projectType) {
+    Version version;
+    try {
+      switch (projectType) {
+        case ProjectType.dart:
+          version = getDartVersion(_getPubspecFile());
+          break;
+        case ProjectType.flutter:
+          version = getFlutterVersion(_getPubspecFile());
+          break;
+      }
+
+      final includeLine =
+          'include: package:yumemi_lints/${projectType.name}/${version.excludePatchVersion}/recommended.yaml';
+      _updateAnalysisOptionsFile(includeLine);
+      return ExitStatus.success;
+    } on FormatException catch (e) {
+      print(e.message);
+      return ExitStatus.error;
+    }
+  }
+
+  File _getPubspecFile() {
+    final pubspecFile = File(
+      path.join(
+        Directory.current.path,
+        'pubspec.yaml',
+      ),
+    );
+
+    if (!pubspecFile.existsSync()) {
+      throw Exception(
+        'The pubspec.yaml file could not be found in the current directory. '
+        'Please ensure that you are in the correct project directory '
+        'or create a pubspec.yaml file if it does not exist.',
+      );
+    }
+
+    return pubspecFile;
+  }
+
+  ProjectType _getProjectType() {
+    final pubspecFile = _getPubspecFile();
+    final lines = pubspecFile.readAsLinesSync();
+    for (final line in lines) {
+      if (line.trim() == 'sdk: flutter') {
+        return ProjectType.flutter;
+      }
+    }
+    return ProjectType.dart;
+  }
+
+  @visibleForTesting
+  Version getFlutterVersion(File pubspecFile) {
+    final yaml = loadYaml(pubspecFile.readAsStringSync()) as YamlMap;
+    final flutterVersion = (yaml['environment'] as YamlMap)['flutter'];
+
+    if (flutterVersion == null) {
+      throw FormatException(
+        'Please put [flutter: Flutter Version] '
+        'under the environment section in pubspec.yaml',
+      );
+    }
+
+    return extractVersion(flutterVersion as String);
+  }
+
+  @visibleForTesting
+  Version getDartVersion(File pubspecFile) {
+    final yaml = loadYaml(pubspecFile.readAsStringSync()) as YamlMap;
+    final dartVersion = (yaml['environment'] as YamlMap)['sdk'];
+
+    if (dartVersion == null) {
+      throw FormatException(
+        'Please put [sdk: Dart Version] '
+        'under the environment section in pubspec.yaml',
+      );
+    }
+
+    return extractVersion(dartVersion as String);
+  }
+
+  @visibleForTesting
+  Version extractVersion(String versionString) {
+    if (versionString.contains('<') && !versionString.contains('>=')) {
+      throw FormatException('Please specify the minimum version.');
+    }
+
+    RegExp regExp;
+    if (versionString.contains('>=')) {
+      regExp = RegExp(r'>=\s*(\d+\.\d+\.\d+)');
+    } else {
+      regExp = RegExp(r's*(\d+\.\d+\.\d+)');
+    }
+    final match = regExp.firstMatch(versionString);
+
+    if (match == null) {
+      throw FormatException(
+        'Version could not be found from pubspec.yaml.',
```

Comment:
![imo-badge](https://img.shields.io/badge/review-imo-skyblue.svg) 
I personally would like to make the message a bit clearer.

![ask-badge](https://img.shields.io/badge/review-ask-yellowgreen.svg) 
How about doing it as follows?
```
The version of Dart or Flutter could not be found in pubspec.yaml. Please ensure that the version is correctly specified for Dart or Flutter.
```

Reply:
The error message was not clear.
Thank you for even suggesting it.
I fixed. 08ae85e93992d2d300894ab0cbde73787bf5e936